HarfangLab/harfanglab/ingest/parser.yml

```
@@ -344,6 +344,7 @@ stages:
             {%- endif -%}
 
           process.parent.executable: "{{json_event.message.process.parent_image}}"
+          process.parent.name: "{{json_event.message.process.parent_image.split('\\\\') | last}}"
```

IMHO, the `basename` filter is more suitable:
```suggestion
          process.parent.name: "{{json_event.message.process.parent_image| basename}}"
```